exercises/x86-64-assembly-2-b/.meta/config.json

```
@@ -0,0 +1,34 @@
+{
+  "solution_files": ["x86_64_assembly_2_b.asm"],
+  "test_file": "x86_64_assembly_2_b_test.c",
+  "tests": [
+    {
+      "name": "test_red",
+      "cmd": "buf"
```

Maybe I misread, but isn't the actual command:

```suggestion
      "cmd": "color_name(buf, RED)"
```

---

Lol, I have no idea what I was thinking when I wrote this :-P